<?xml version="1.0"?>
<launch>
	<node name="tm_driver" pkg="tm_driver" type="tm_driver" output="screen">
		<param name="robot_ip_address" type="str" value="169.254.77.215" />
	</node>
	 <node name="loop_trajectory" pkg="obstacle_avoidance" type="loop_trajectory" respawn="false" output="screen" />
	<node name="obstacle_avoidance_naive3" pkg="obstacle_avoidance" type="obstacle_avoidance_naive3" respawn="false" output="screen" />
</launch>
